src/Services/Users/index.js

```diff
@@ -32,6 +32,23 @@ class Users {
             .catch((error) => error);
     }
 
+    static getPlainUsers() {
+        return http
+            .get('https://reqres.in/api/users?page=1')
+            .then((res) => {
+                const newList = [];
+                res.data.forEach((element) => {
+                    newList.push([element.id,
+                        element.email,
+                        element.first_name,
+                        element.last_name,
+                        element.avatar]);
+                });
```

Comment:
Two issues to consider here:

If the element is not used for anything other than pushing into the array, you might use the following shorthand:
forEach( ({ id, email, first_name, last_name, avatar }) => newList.push([ id, email, first_name, last_name, avatar ]) );

Either way, by destructuring the object and setting its properties as array elements, you are obfuscating its usage for whoever uses this list - which is a bad thing.
I'd suggest you simply pass the array of elements and let whoever uses it define how it should be formatted.

Reply:
Passing just an array from User Service to Table Component